common/common.go

```diff
@@ -135,3 +135,13 @@ func SupportsOverlay() bool {
 	}
 	return false
 }
+
+// RunningSystemd returns whether the host runs systemd
```

Comment:
This should go upstream into go-systemd - https://github.com/coreos/go-systemd/issues/88